GA4 identify call (#822)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/google-analytics-4/setting-up-google-analytics-4-in-rudderstack.mdx

</ul>
</div>

### Identify 

<div class="infoBlock">

> The settings in this section are only applicable for the <strong>cloud mode</strong>.

Reviewer: ```suggestion
The settings in this section are only applicable for the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode"><strong>cloud mode</strong></Link>.
```

</div>

<img src="../../../assets/event-stream-destinations/google-analytics-4-connection-settings-1.png" alt="Google Analytics 4 Connection Settings in RudderStack" />

- **Enable Server Side Identify**: Enable this setting and configure the following options to make an <Link to="/event-spec/standard-events/identify/">`identify`</Link> call:

>   - **Trait to differentiate between new or existing user**: Enter a trait key to determine whether a user is new or an existing user.

Reviewer: ```suggestion
  - **Trait to differentiate between new or existing user**: Enter a trait key to determine a new or an existing user.
```

  - **Convert identify event to Login or Signup event**: Enable this setting to convert the `identify` event to a <Link to="https://developers.google.com/analytics/devguides/collection/ga4/reference/events#login">`login`</Link> event in case of an existing user or a <Link to="https://developers.google.com/analytics/devguides/collection/ga4/reference/events#sign_up">`sign_up`</Link> event in case of a new user.

>   <div class="infoBlock">

Reviewer: ```suggestion
  <div class="warningBlock">
```

>   It is recommended to enable this setting in order to send the <code class="inline-code">identify</code> call successfully.

Reviewer: ```suggestion
If <strong>Enable Server Side Identify</strong> is enabled in the dashboard, then this setting too <strong>must</strong> be enabled to send the <code class="inline-code">identify</code> call successfully.
```

>   </div>

Reviewer: As per the Slack conversation, it seems that this should be a condition rather than a recommendation, IMO. Suggested accordingly.

  - **Value - traits for lead generation**: Enter the value key to pass the `value` parameter for the `generate_lead` event. 
  - **Currency - traits for lead generation**: Enter the currency key to pass the `currency` parameter for the `generate_lead` event. 

  <div class="infoBlock">
  Refer to the <Link to="/destinations/streaming-destinations/google-analytics-4/google-analytics-4-cloud-mode/#identify"><code class="inline-code">identify</code></Link> call payload for more information on sending <code class="inline-code">identify</code> events to GA4 via cloud mode.
  </div>

### Native SDK

<div class="infoBlock">

> The settings in this section are only applicable for the< strong>device mode</strong>.

Reviewer: ```suggestion
The settings in this section are only applicable for the <Link to="/destinations/rudderstack-connection-modes/#device-mode"><strong>device mode</strong></Link>.
```
